00-master.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/bash
```

Comment:
As we're integrating with [DO-CCM](https://github.com/digitalocean/digitalocean-cloud-controller-manager) (reference: [`deploy.tf`](https://github.com/kubernetes-digitalocean-terraform/kubernetes-digitalocean-terraform/pull/51/files#diff-c991e59a6cc5ca08ecab0b9ee6a45a74R204)), maybe we have to set the `--cloud-provider=external` flag for `kubelet` service.

I'm not sure is this required, by it can be done by adding the following line to the `/etc/systemd/system/kubelet.service.d/10-kubeadm.conf` file:
```
Environment="KUBELET_EXTRA_ARGS=--cloud-provider=external"
```
After that, you have to reload the daemon and restart the `kubelet` service:
```
systemctl daemon-reload
systemctl restart kubelet
```

**Important warning:** Make sure to test the changes. I had several problems that I wasn't able to solve involving Flannel and `cloud-provider` flag.

Also, do we want to add `--node-ip` flag to `kubelet`, like for workers?

@andrewsykim @bhcleek: Is this needed?

Reply:
> As we're integrating with DO-CCM (reference: deploy.tf), maybe we have to set the --cloud-provider=external flag for kubelet service.

I'm pretty sure `kubeadm` use [TLS bootstrapping](https://kubernetes.io/docs/admin/kubelet-tls-bootstrapping/) which [doesn't work with CCM](https://kubernetes.io/docs/tasks/administer-cluster/running-cloud-controller/#chicken-and-egg). See https://github.com/kubernetes/kubernetes/issues/55633 for the current status.